CONTRIBUTING.md

```diff
@@ -88,6 +88,172 @@ One of the best ways we can keep fastlane an approachable, stable, and dependabl
 
 Writing a custom action is an easy way to extend the capabilities of fastlane. Actions that make good candidates for inclusion in the fastlane codebase are **flexible** and apply to **many projects, teams, and development setups**. Before working to contribute your custom action to fastlane, consider whether it is likely to solve a problem that many developers have. If not, it can still provide value for your fastlane environment! Check out the documentation for creating [local action extensions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/README.md#extensions).
 
+## Getting started
```

Comment:
I wonder if we might want to put this up a little bit? I wouldn't want to read all those docs before I can actually see how I can contribute. cc @mfurtak